JobRunner/DockerRunner.py

```diff
@@ -21,30 +18,41 @@ def __init__(self, logger=None):
         self.logger = logger
         self.containers = []
         self.threads = []
+        self.log_interval = 1
 
     def _sort_logs(self, sout, serr):
         """
         This is an internal function to sort and interlace output for NJS.
         This is not fully implemented yet and sould be rethought.
         """
-        # TODO: Fix sorting
 
-        lines = []
+        tslines = dict()
```

Comment:
What does tslines stand for?

Reply:
time stamped lines.  We could make it longer.